package.json

```diff
@@ -84,12 +84,12 @@
     "react-hook-form": "^7.34.0",
     "react-i18next": "^11.18.3",
     "react-leaflet": "^4.0.1",
-    "react-medium-image-zoom": "^4.4.3",
+    "react-medium-image-zoom": "^5.0.1",
```

Comment:
Implementation needs to be adapted. Zooming the images on the blog post pages doesn't work anymore.

Reply:
Maybe it is a cache thing on your end, because it is working for me (Tested in Safari & Firefox)
![image-zoom](https://user-images.githubusercontent.com/3493187/183385462-3d4016f7-d6e1-49df-9bcb-c0b0027d3cb3.gif)

Reply:
Ahh, you are right. After checking again, it's now also working for me :)